pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -484,15 +485,17 @@ def test_to_datetime_tz_psycopg2(self, cache):
         # dtype coercion
         i = pd.DatetimeIndex([
             '2000-01-01 08:00:00+00:00'
-        ], tz=psycopg2.tz.FixedOffsetTimezone(offset=-300, name=None))
+        ])
+        i = i.tz_convert(psycopg2.tz.FixedOffsetTimezone(offset=-300,
+                                                         name=None))
         assert is_datetime64_ns_dtype(i)
 
         # tz coerceion
         result = pd.to_datetime(i, errors='coerce', cache=cache)
         tm.assert_index_equal(result, i)
 
         result = pd.to_datetime(i, errors='coerce', utc=True, cache=cache)
-        expected = pd.DatetimeIndex(['2000-01-01 13:00:00'],
+        expected = pd.DatetimeIndex(['2000-01-01 08:00:00'],
```

Comment:
reason for this change?

Reply:
The `+00:00` in `i` above required a `tz_convert` post construction while I think the original intention was to `tz_localize` it to the passed `psycopg.tz` in the constructor (leading to this change)

But to reflect the intension of the original test, I just removed the `+00:00` instead